applications/luci-app-hd-idle/htdocs/luci-static/resources/view/hd_idle.js

```diff
@@ -2,43 +2,89 @@
 'require form';
 'require fs';
 'require view';
+'require uci';
+
+function disk(devs, option, section_id) {
+	var v = uci.get('hd-idle', section_id, 'disk') || '';
+	var e = devs.findIndex(function(itm) { return itm.name == v; } );
```

Comment:
Maybe use `devs.find()` instead of `devs.findIndex()` ? Saves the extra `devs[e]` and `e != -1` checks later on.

Reply:
I changed it to `.find()`, however I still have a `!= undefined`.